create_aio_app/main.py

```diff
@@ -15,6 +16,22 @@
 echo = partial(click.echo, err=True)
 
 
+def print_commands(commands):
+    print('\n\n')
+    for name, description in commands.items():
+        echo(click.style(name + ' - ', fg='yellow'))
+        echo(click.style(description, fg='bright_green'))
+
+
+def show_commands():
+    echo(click.style('\n    Common', fg='cyan'))
+    print_commands(COMMON)
+    echo(click.style('\n    Linters', fg='cyan'))
+    print_commands(LINTERS)
+    echo(click.style('\n    Database', fg='cyan'))
+    print_commands(DATABASE)
+
+
```

Comment:
I think that we have better way to do this. We actually have feature which show all commands from `makefile`.

```
make help
```

two reasons to use that:

- don't repeat yourself
- auto generation of output. `make help` generate automation output and we don't need update this each time when we change `makefile`.

So, I propose use some like this:

```python
import os

try:
    os.system('make help')
except Exception:
    ...
```
We need to wrap this to `try` block because not each `os` has `make`. 

what do u think?

Reply:
I agree it's a better solution 👍